pkgs/development/python-modules/pydantic/default.nix

```diff
@@ -43,6 +69,18 @@ buildPythonPackage rec {
     export HOME=$(mktemp -d)
   '';
 
+  # Must include current directory into PYTHONPATH, since documentation
+  # building process expects "import pydantic" to work.
+  preBuild = ''
+    PYTHONPATH=$PWD:$PYTHONPATH make docs
+  '';
+
+  # Layout documentation in same way as "sphinxHook" does.
```

Comment:
Why is sphinxHook doing that?

Reply:
Okay, we can continue discussing pros and cons in https://github.com/NixOS/nixpkgs/pull/175650